tests/e2e/e2e_test.go

```diff
@@ -149,6 +150,22 @@ func (ts *E2ETestSuite) startServer(extraArgs ...string) {
 	}
 }
 
+func (ts *E2ETestSuite) startTask(taskFile string) types.JobID {
```

Comment:
consistency in naming concepts

```suggestion
func (ts *E2ETestSuite) startJob(descriptorFile string) types.JobID {
```